stable/fairwinds-insights/values.yaml

```diff
@@ -81,6 +81,13 @@ hubspotCronjob:
       cpu: 80m
       memory: 128Mi
   schedules: []
+  # -- Optional read replica configuration
+  readReplica:
```

Comment:
we might want to reuse this in other places--maybe we should put it under `postgres`

Reply:
I thought it may hurt readability to scope it to `postgres`, because if you're reading the chart it's not clear what the read replicas are for. I suppose I could make this change and address that in the README if it works for you

Reply:
Yeah good point--we could pull it out as a top-level option too. I just don't want to scope it to `hubspot` and then decide it's useful somewhere else